src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -457,16 +458,20 @@ static void window_server_list_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi
             colour = COLOUR_MOSS_GREEN;
         }
 
+        screenCoords.x = 3;
+        screenCoords.y += 3;
+
```

Comment:
```suggestion
        screenCoords.x = 3;

```
Let's not update `y` here because we then have to decrement afterwards